darts/models/theta.py

```diff
@@ -41,8 +42,8 @@ def __init__(self,
         seasonality_period
             User-defined seasonality period. If not set, will be tentatively inferred from the training series upon
             calling `fit()`.
-        mode
-            Type of seasonality. Either "additive" or "multiplicative".
+        season_mode
+            Type of seasonality. Must be a Season Enum member. Defaults to `Season.MULTIPLICATIVE`.
```

Comment:
I think for all the enums we should either link to the possible values, or list them (including `Season.NONE`).